src/components/components.js

```diff
@@ -7,4 +7,11 @@ export default angular.module('myModule.components', [])
   .component('myMap', myMap)
   .component('myChart', myChart)
   .component('myTable', myTable)
-  .component('myToolbar', myToolbar);
+  .component('myToolbar', myToolbar)
+  .config(WidgetsProvider => {
+    'ngInject';
+
+    WidgetsProvider.register('myMap', myMap);
+    WidgetsProvider.register('myChart', myChart);
+    WidgetsProvider.register('myTable', myTable);
```

Comment:
Removed from what? What distinguishes widget components from toolbar components? Does this mean I can't search widgets for toolbar?